vue/components/ui/molecules/carousel/carousel.vue

```diff
@@ -0,0 +1,283 @@
+            v-bind:color="arrowsColor"
+            v-bind:icon="'chevron-left'"
+            v-bind:width="36"
+            v-bind:height="36"
+            v-show="arrowsVisibility"
+            v-on:click="previous"
+        />
+        <transition-group class="slide-container" tag="div" name="fade">
+            <div
+                class="slide"
+                v-for="(image, index) in images"
+                v-show="index === imageIndexData"
+                v-bind:key="`${image}-${index}`"
+            >
+                <div class="slide-content">
+                    <image-ripe class="slide-image" v-bind:src="image" v-on:load="onLoadImage" />
+                </div>
+            </div>
+        </transition-group>
+        <icon
+            class="arrow arrow-next"
+            v-bind:color="arrowsColor"
+            v-bind:icon="'chevron-right'"
+            v-bind:width="36"
+            v-bind:height="36"
+            v-show="arrowsVisibility"
+            v-on:click="next"
+        />
+    </div>
+</template>
+
+<style lang="scss" scoped>
+.carousel {
+    height: 100%;
+    overflow: hidden;
+    position: relative;
+    user-select: none;
+    width: 100%;
+}
+
+.carousel .arrow {
+    cursor: pointer;
+    position: absolute;
+    top: calc(50% - 18px);
+    transition: scale ease-in-out 0.1s, transform ease-in-out 0.1s;
+    z-index: 1;
+}
+
+.carousel .arrow:hover {
+    scale: 1.3;
+}
+
+.carousel .arrow:active {
+    scale: 1.4;
+}
+
+.carousel .arrow-previous {
+    left: 0px;
+}
+
+.carousel .arrow-next {
+    right: 0px;
+}
+
+.carousel .arrow-previous:active {
+    transform: translate(-5px);
+}
+
+.carousel .arrow-next:active {
+    transform: translate(5px);
+}
+
+.carousel .slide-container {
+    height: 100%;
+    position: relative;
+    text-align: left;
+    white-space: nowrap;
+    width: 100%;
+}
+
+.carousel .slide-container .slide {
+    cursor: pointer;
+    display: inline-block;
+    height: 100%;
+    position: absolute;
+    text-align: center;
+    vertical-align: top;
+    width: 100%;
+}
+
+.carousel .slide-container .slide .slide-content {
+    align-items: center;
+    display: flex;
+    height: 100%;
+    justify-content: center;
+    padding: 0px 5px 0px 5px;
+    width: 100%;
+}
+
+.carousel .slide-container .slide .slide-content .image {
+    height: 100%;
+    object-fit: contain;
+    pointer-events: none;
+    width: 100%;
+}
+
+.fade-enter-active,
+.fade-leave-active {
+    transition: opacity 0.5s;
+}
+
+.fade-enter,
+.fade-leave-to {
+    opacity: 0;
+}
+</style>
+
+<script>
+export const Carousel = {
+    name: "carousel",
+    props: {
+        /**
+         * Set of images to be displayed in the carousel slider.
+         */
+        images: {
+            type: Array,
+            default: () => []
+        },
+        /**
+         * The index of the image to be displayed first.
+         */
+        imageIndex: {
+            type: Number,
+            default: 0
+        },

```

Comment:
Don't bind this component to images. Even if the default implementation is a `image-ripe`
```suggestion
        /**
         * Set of images to be displayed in the carousel slider.
         */
        items: {
            type: Array,
            default: () => []
        },
        /**
         * The index of the image to be displayed first.
         */
        value: {
            type: Number,
            default: 0
        },

```